assets/samplesheet.csv

```diff
@@ -1,3 +1,3 @@
 sample,fastq_1,fastq_2
-SAMPLE_PAIRED_END,/path/to/fastq/files/AEG588A1_S1_L002_R1_001.fastq.gz,/path/to/fastq/files/AEG588A1_S1_L002_R2_001.fastq.gz
-SAMPLE_SINGLE_END,/path/to/fastq/files/AEG588A4_S4_L003_R1_001.fastq.gz,
+SAMPLE1_PE,https://github.com/nf-core/test-datasets/raw/mag/test_data/test_minigut_R1.fastq.gz,https://github.com/nf-core/test-datasets/raw/mag/test_data/test_minigut_R2.fastq.gz
```

Comment:
We actually don't have our samples for the small dataset, only for the full test. Do you think is better to have our own samples? Because these samples work pretty well for us. Should I ask an opinion on Slack?

Reply:
I am not against it that you use the same data. It just seems risky: if for whatever reason those files in nf-core/mag are not available any more your test fails as well. So my suggestion is to copy that data to you. But that would inflate storage, so probably ask in slack for more opinions on that.

Reply:
I decided to upload the files to test-dataset repo since we shouldn't rely on other pipelines (they could change the files anytime as you said).  just waiting for the approval and then I will update the samplesheet.csv

Reply:
I just updated the file